magpie/run/magpie-run-project-alluxio

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+########################################################################
+#  Project Magpie. For details, see https://github.com/llnl/magpie.
+#
+#  Copyright (C) 2020 Intel Corporation. All rights reserved.
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License version
+#  2 as published by the Free Software Foundation.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  Aleksander Kantak <aleksander.kantak@intel.com>
+########################################################################
+
+# These are functions to be called by magpie-run
+
+source ${MAGPIE_SCRIPTS_HOME}/magpie/exports/magpie-exports-submission-type
+source ${MAGPIE_SCRIPTS_HOME}/magpie/exports/magpie-exports-dirs
+source ${MAGPIE_SCRIPTS_HOME}/magpie/exports/magpie-exports-user
+source ${MAGPIE_SCRIPTS_HOME}/magpie/lib/magpie-lib-defaults
+source ${MAGPIE_SCRIPTS_HOME}/magpie/lib/magpie-lib-node-identification
+source ${MAGPIE_SCRIPTS_HOME}/magpie/lib/magpie-lib-run
+source ${MAGPIE_SCRIPTS_HOME}/magpie/lib/magpie-lib-log
+source ${MAGPIE_SCRIPTS_HOME}/magpie/lib/magpie-lib-paths
+
+Magpie_run_start_alluxio () {
+    if [ "${ALLUXIO_SETUP}" == "yes" ] && [ "${magpie_run_prior_startup_successful}" == "true" ]
+    then
+        cd ${ALLUXIO_HOME}
+
+        if [ ${MAGPIE_JOB_TYPE} != "setuponly" ]
+        then
+
+            if [ "${ALLUXIO_DATA_CLEAR}" == "yes" ]
+            then
+                echo "Formatting Alluxio"
+                ${alluxiocmdprefix}/alluxio format
+            fi
+
+            echo "Starting alluxio"
+            ${alluxiocmdprefix}/alluxio-start.sh all ${ALLUXIO_RAMFS_MOUNT_OPTION}
+
+        fi
+
+        echo "*******************************************************"
+        echo "*"
+        echo "* Alluxio Information"
+        echo "*"
+        echo "* You can view your Alluxio status by launching a web browser and pointing to ..."
+        echo "*"
+        echo "* Alluxio UI: http://${ALLUXIO_MASTER_NODE}:${default_alluxio_master_web_port}"
+        echo "*"
+        echo "*"
+        echo "* You can use your Alluxio storage pointing application, ex. Spark to ..."
```

Comment:
this doesn't sound right.  "You can point your Alluxio storage application, ex. Spark, to ..."

??